style.css

```diff
@@ -1,5 +1,7 @@
+
 #map {
-  height: 100%;
+  height: 500px;
```

Comment:
How does this look on different screen sizes? Does this fill up the page?

Reply:
We can always adjust later. This was pretty arbitrary. It definitely fits phones but desktop is where this will be a bit short. 100% just didnt do anything